qt/aqt/browser.py

```diff
@@ -194,23 +186,18 @@ def flags(self, index):
     def search(self, txt: str) -> None:
         self.beginReset()
         self.cards = []
-        exception: Optional[Exception] = None
         try:
             ctx = SearchContext(search=txt, browser=self.browser)
             gui_hooks.browser_will_search(ctx)
             if ctx.card_ids is None:
-                ctx.search = self.browser.normalize_search(ctx.search)
-                ctx.card_ids = self.col.find_cards(ctx.search, order=ctx.order)
+                ctx.card_ids = list(self.col.find_cards(ctx.search, order=ctx.order))
```

Comment:
Quick note: I had to explicitly make card_ids a list because it would be passed as a `protobuf.internal.containers.RepeatedScalarFieldContainer` which doesn't support all features lists do.
I have no idea why this suddenly became necessary and if it indicates I did anything wrong.

Reply:
What was the traceback?

Reply:
If it's just a typechecking issue, self.cards could probably be declared as a Sequence instead of a List

Reply:
The problem was in browser.py/focusCid() because it doesn't have an index method. But somehow it's working on the master branch.
I didn't get an error because of the universal exception handling.

Reply:
It might have something to do with calling `model.search` multiple times. On the main branch, current card search is only supported with a closed browser, so `self.model.cards.index(cid)` is only called after the very first search.

Reply:
Ah, makes sense. That routine will only need to run with a single note shown, so perhaps we could just use list() inside focusCid, and avoid having to build a separate list in other cases.

Reply:
(the catch-all exception there is also nasty)